deckhouse-controller/crds/doc-ru-module.yaml

```diff
@@ -4,7 +4,7 @@ spec:
       schema:
         openAPIV3Schema:
           description: |
-            Репрезентация модуля
+            Репрезентация модуля.
```

Comment:
```suggestion
            Описывает состояние модуля в кластере. Объект Module будет создан автоматически после настройки источника модулей ([ModuleSource](#modulesource)) и успешной синхронизации.
```

Reply:
@miklezzzz Please review.

Reply:
I don't mind, but I have to admit that this change hardly belongs to the Pr :)